blog/_src/posts/2017-06-24-turing-award-50.md

```diff
@@ -0,0 +1,732 @@
+    Title: Quotes and Stories from "Turing 50"
+    Date: 2017-06-24T20:00:52
+    Tags: dear diary, by Ben Greenman
+
+The ACM recently hosted [a celebration of 50 years of the A.M. Turing award](https://www.acm.org/turing-award-50).
+These are some notes and thoughts from the event,
+including how Fred Brooks once rented a bus,
+Don Knuth's outrageous implementation of batch processing,
+and Judea Pearl's theory of homo sapiens.
+
+<!-- more -->
+
+**Conventions / Disclaimers:**
+
+- Italic text is paraphrased, may be incorrect, and may be incorrectly attributed.
+  Make sure to watch the ACM's live stream before quoting anything here!!!
+
+- Section-breaks are labeled as "panel", "talk", "question", etc.
+
+- This is intentionally "bad writing" in the Peter Lee sense (see below)
+  --- primarily "what I saw" and very little about "what I thought and felt".
+  A summary in my own words just wouldn't do justice to the panelists.
+
+- The "Augmented Reality" session was my favorite.
+
+### Opening Remarks
+
+#### _Alan Turing is with us today_
+
+At the start of the event, the [emcee](http://users.ecs.soton.ac.uk/wh/) unvelied a bronze bust of Alan Turing.
+This statue was on display at center stage during the whole event.
+
+It's a good sculpture and it's good we remember Alan Turing, but I'm sad
+ that the ACM would encourage this kind of idol-worship.
+
+
+### talk: Impact of Turing Recipients' Work
+
+Speaker: Barbara Liskov
+
+- _the first awards recognized achievements in the standard fields of theory, AI, and systems_
+- _hostile environment around the first awards, trepidation about future awards_
+- _with Unix, Ritchie and Thompson got the design right_
+- _Niklaus Wirth said, 'If I understood how important Pcode was, I would have
+   spent more time designing it'_
+
+#### thoughts
+
+What is "systems" --- does that even have a definition?
+And Unix is definitely NOT an example of a "right design"; rather it's a
+ landmark of [worse is better](https://www.dreamsongs.com/WorseIsBetter.html) design.
+
+### panel: Advances in Deep Neural Networks
+
+Panelists: Judea Pearl (JP) Michael Jordan (MJ) Fei-Fei Li (FL) Stuart Russell (SR)
+  Ilya Sutskever (IS) Raquel Urtasun (RU).
+
+- _please welcome Judea Pearl and his panel_
+- SR: _I work in all areas of AI except for deep learning_
```

Comment:
Would it maybe be possible to expand the abbreviations to full names now? I find it a bit hard to read this way.